logger/logger_logrus_test.go

```diff
@@ -25,14 +31,38 @@ var _ = Describe("logger tests", func() {
 		Expect(hook.LastEntry().Level).To(Equal(githubLogrus.InfoLevel))
 		Expect(hook.LastEntry().Message).To(Equal("sample_activity_log"))
 
-		logger.AnomalyLog("sample_anomaly_log", data)
+		logger.ErrorLog("sample_anomaly_log", nil, data)
 		Expect(len(hook.Entries)).To(Equal(3))
 		Expect(hook.LastEntry().Level).To(Equal(githubLogrus.ErrorLevel))
 		Expect(hook.LastEntry().Message).To(Equal("sample_anomaly_log"))
 
-		logger.AnomalyLogError("sample_anomaly_log_2", errors.New("error message"), nil)
+		logger.ErrorLog("sample_anomaly_log_2", errors.New("error message"), nil)
 		Expect(len(hook.Entries)).To(Equal(4))
 		Expect(hook.LastEntry().Level).To(Equal(githubLogrus.ErrorLevel))
 		Expect(hook.LastEntry().Message).To(Equal("sample_anomaly_log_2"))
+
+		logger.Log(ERROR, "http: TLS handshake error from 0.0.0.0:1: EOF", nil)
+		Expect(len(hook.Entries)).To(Equal(5))
+		Expect(hook.LastEntry().Level).To(Equal(githubLogrus.ErrorLevel))
+		Expect(hook.LastEntry().Message).To(Equal("http: TLS handshake error from 0.0.0.0:1: EOF"))
 	})
+
+	Context("Tls handshake error", func() {
+		DescribeTable("supresses",
+			func(envValue string) {
+				os.Setenv("SUPPRESSE_TLS_HANDSHAKE_ERROR_LOGGING", envValue)
+				logger, hook := NoOpLogger()
+				logger.Log(ERROR, "http: TLS handshake error from 0.0.0.0:1: EOF", nil)
+				Expect(hook.Entries).To(BeEmpty())
+				logger.Log(ERROR, "Random error", nil)
+				Expect(len(hook.Entries)).To(Equal(1))
+				Expect(hook.LastEntry().Message).To(Equal("Random error"))
+
+			},
+			Entry("for env variable true", "true"),
+			Entry("for env variable TRUE", "TRUE"),
+			Entry("for env variable 1", "1"),
+		)
+	})
+
```

Comment:
Maybe add a case where the env var is set to `true` and we see it doesn't suppress other messages?